week2/2-website/css/style.css

```diff
@@ -1,19 +1,261 @@
 
  /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+*{
 
+    padding:0;
+    margin:0;
+    box-sizing: border-box;
+}
 body {
     font-family: 'Roboto', sans-serif;
     -webkit-font-smoothing: antialiased;
+    height: 100vh;
+}
+
+
+.devices{
+    
+    display:flex;
+    width: 60%;
+    margin:auto;
+    justify-content: space-around;
+
 }
+.devices li{
 
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
- */
+    text-align: center;
+}
+header {
+width: 100%;
+background-color: white;
+height: 6vh;
+}
+.content-nav {
+    align-items: center;
+    height: 100%;
 
+}
+.links {
+ width: 80% ;
+ justify-content: flex-end;
+ height: 100%;
+ align-items: center;
+}
+.rapper {
+    border:1px solid green;
```

Comment:
Is there a reason there's a green border to this? Did you forget to push another commit where this is removed?